ElectronClient/gui/NoteText.jsx

```diff
@@ -1612,18 +1612,30 @@ class NoteTextComponent extends React.Component {
 		menu.popup(bridge().window());
 	}
 
+	async updateFolderTreeOnSearch(folder) {
+		while (folder && folder.parent_id) {
+			this.props.dispatch({
+				type: 'FOLDER_TOGGLE',
+				id: folder.parent_id,
+			});
+			folder = await Folder.load(folder.parent_id);
+		}
+	}
+
 	createToolbarItems(note, editorIsVisible) {
 		const toolbarItems = [];
 		if (note && this.state.folder && ['Search', 'Tag', 'SmartFilter'].includes(this.props.notesParentType)) {
 			toolbarItems.push({
 				title: _('In: %s', substrWithEllipsis(this.state.folder.title, 0, 16)),
 				iconName: 'fa-book',
-				onClick: () => {
+				onClick: async () => {
```

Comment:
Why async?

Reply:
Sorry about that. Earlier I retrieved the Folder directly inside the onClick handler. Later I moved it to a function. Forgot to remove it then.